packages/ui/useControlledState/useControlledState.js

```diff
@@ -4,10 +4,18 @@ import React from 'react';
 // and is memoized by the onChange provided by useState
 function createHandler () {
 	return (onChange) => (value) => {
-		onChange(prevState => ({
-			value: typeof value === 'function' ? value(prevState.value) : value,
-			controlled: prevState.controlled
-		}));
+		onChange(prevState => {
```

Comment:
We might want to do a bit more with `useControlledState` later. Since a `controlled` instance can't become uncontrolled and a `controlled` instance always uses the prop values, we don't need `onChange` to do anything. We could cache a no-op function and return it instead when `controlled` and, when uncontrolled, simplify this logic to not even call the `onChange` hook when the value hasn't changed.

Reply:
Possible impl

https://github.com/enactjs/enact/tree/feature/PLAT-112393-ryan

Reply:
I had considered that but didn't do it.  If you've put it together, though, let me take a look.